metaci_cli/cli/config.py

```diff
@@ -25,7 +25,7 @@ def dbm_cache():
     """
     config_dir = os.path.join(
         os.path.expanduser('~'),
-        YamlGlobalConfig.config_local_dir,
+        BaseGlobalConfig.config_local_dir,
```

Comment:
sigh I know this isn't a today thing, but this whole check latest version stuff is messy (I know, I wrote it!) and has a better impl over in CCI.